ingestion/src/metadata/data_quality/validations/column/sqlalchemy/columnValuesToBeUnique.py

```diff
@@ -57,7 +58,18 @@ def _run_results(self, metric: Metrics, column: Column) -> Optional[int]:
         )  # type: ignore
 
         try:
-            self.value = dict(self.runner.dispatch_query_select_first(count, unique_count.scalar_subquery().label("uniqueCount")))  # type: ignore
+            if self.runner.dialect == Dialects.Oracle:
+                query_group_by_ = [literal_column("2")]
+            else:
+                query_group_by_ = None
```

Comment:
That is a bit hacky, thinking we can revisite the implementation if we start seeing it happening across more than 1 dialect -- so far only reported in Oracle and implementation has been there for 2+ releases.